src/validation/fields/helper/github-handles.validation.helper.ts

```diff
@@ -0,0 +1,18 @@
+import { ErrorMessages } from '../../error.messages';
+
+export const validateGithubHandles = (value: string) => {
+    if (!value) {
+        throw new Error(ErrorMessages.GIT_HANDLE);
+    }
+
+    const handles = value.split(',').map(handle => handle.trim());
+    const githubHandleRegex = /^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*$/;
+
+    for (const handle of handles) {
+        if (!githubHandleRegex.test(handle) || handle.length > 39 || handle.length < 1) {
```

Comment:
39 here is a bit of a "magic number" here. I would make a variable to describe what it's purpose is to give context: `const githubHandleMaxLength = 39`